### PR TITLE
Make the number of repos returned configurable

### LINKED ISSUE #4: Make the number of repos returned configurable with an environment variable
Default it to 6 if not specified.

### PR DESCRIPTION
**Pull request by yeskunall**
Hey 👋

Currently `example-response.json` contains 6 repos. My test case doesn't _really_ add anything meaningful. I'd like to add more repos to the response file. Although I'm fairly certain my additions work as expected, but just to be sure.

So, when you have time, can you review the changes and also give me something to add to the file so I can test it against my changes better?

Thanks ❤️ 

---

Fixes #4

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const controlAccess = require('control-access');
 const token = process.env.GITHUB_TOKEN;
 const username = process.env.GITHUB_USERNAME;
 const origin = process.env.ACCESS_ALLOW_ORIGIN;
+const maxRepos = process.env.MAX_REPOS ? process.env.MAX_REPOS : 6;
```
Comment (yeskunall) [inline index.js]: Think this should be Number(process.env.MAX_REPOS) instead. :thinking:

Comment (yeskunall) [inline index.js]: I think `process.env.MAX_REPOS` needs to be wrapped in a `Number`. :thinking: shit, don't know how I missed that after fixing the test case. Hit me with a review, and I'll fix it once I hear back from you.

Comment (mathdroid) [reply index.js]: Yes it needs to be converted to `number`. Also the use of ternary operator is a bit redundant here. I would use

```js
const maxRepos = Number(process.env.MAX_REPOS) || 6
```

**Comment by sindresorhus** [comment]
Looks good when you fix: https://github.com/sindresorhus/gh-latest-repos/pull/6/files#r144708597

And update https://deploy.now.sh/?repo=https://github.com/sindresorhus/gh-latest-repos&env=GITHUB_TOKEN&env=GITHUB_USERNAME&env=ACCESS_ALLOW_ORIGIN and the below instruction to include the environment variable.

**Comment by sindresorhus** [comment]
Great. Thank you @yeskunall :)

**Pull request merged**